src/python.cc

```diff
@@ -234,7 +238,7 @@ TRITONSERVER_Error *ModelInstanceState::CreatePythonInterpreter() {
     subinterpreter_commandline[5] = pymodule_path_.c_str();
     subinterpreter_commandline[7] = name_.c_str();
     if (execvpe(subinterpreter_commandline[0],
```

Comment:
Since you are just passing in the existing env you can just use execvp and not need to use "environ"

Reply:
fixed. Also, @tanmayv25 pointed out the reason the formatting was inconsistent. I fixed that and reformatted the whole document.